lib/selection-count-view.coffee

```diff
@@ -1,15 +1,28 @@
+_ = require 'underscore-plus'
+
 class SelectionCountView extends HTMLElement
+  toolTipDisposable: null
```

Comment:
Actually, I stole some lines from [here](https://github.com/atom/deprecation-cop/blob/master/lib/deprecation-cop-status-bar-view.coffee#L16). Should we also delete here ?